README.md

```diff
@@ -120,6 +120,8 @@ Eg: `curl ... | NVM_DIR="path/to/nvm"`. Ensure that the `NVM_DIR` does not conta
 
 - The installer can use `git`, `curl`, or `wget` to download `nvm`, whichever is available.
 
+- You can instruct the installer to not edit your shell config (for example if you already get completions via a [zsh nvm plugin](https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/nvm)) by setting `PROFILE=/dev/null` before running the `install.sh` script. Here's an example one-line command to do that: `PROFILE=/dev/null bash -c 'curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash'`
```

Comment:
why do you need `bash -c` here?

Reply:
`bash -c` is necessary to make `$PROFILE` visible to the other side of the pipe.

```sh
> cat install.sh
echo profile is $PROFILE

> PROFILE=/dev/null ./install.sh
profile is /dev/null

> PROFILE=/dev/null cat install.sh | bash
profile is

> PROFILE=/dev/null bash -c 'cat install.sh | bash'
profile is /dev/null
```

Reply:
aha, thanks